tests/acceptance/stepDefinitions/generalContext.js

```diff
@@ -134,7 +134,7 @@ const setTrustedServer = function (url) {
   const body = new URLSearchParams()
   body.append('url', userSettings.replaceInlineCode(url))
   const headers = httpHelper.createAuthHeader(client.globals.backend_admin_username)
-  const postUrl = `${backendHelper.getCurrentBackendUrl()}/ocs/v2.php/apps/testing/api/v1/trustedservers?format=json`
+  const postUrl = path.join(backendHelper.getCurrentBackendUrl(), '/ocs/v2.php/apps/testing/api/v1/trustedservers?format=json')
```

Comment:
in this file, `path` is a node-module, not the helper.

Reply:
fixed